test/powershell/SDK/Breakpoint.Tests.ps1

```diff
@@ -115,6 +108,14 @@ Describe 'Breakpoint SDK Unit Tests' -Tags 'CI' {
             }
         }
 
+        It 'Doesn''t manipulate any breakpoints in the default runspace' {
+            # Issue https://github.com/PowerShell/PowerShell/issues/10167 fix:
+            # Ensure that breakpoints were not created in the default runspace.
+            # Prior to this issue being fixed, breakpoints with the same id
+            # would be created or updated in the default runspace.
+            $host.Runspace.Debugger.GetBreakpoints() | Should -BeNullOrEmpty
```

Comment:
test cases should be independent.

Reply:
But this is not blocking.

Reply:
@TravisEz13 A question about this for my own knowledge: by independent do you mean that the specific test for that issue should be in a different `Describe` or `Context` invocation? Or do you mean that the comments shouldn't be there, because they infer a dependency between that test and the issue it resolves?